package ch1_collection.exception;

public class ExceptionEx1 {

    public static void main(String[] args) {

        int result = 10 / 0;

        /*
        try {
            // 예외가 발생할 수 있는 코드
            int result = 10 / 0;
        } catch (ArithmeticException e) {
            // 예외 처리 코드
            System.out.println("0으로 나눌 수 없습니다: " + e.getMessage());
        } finally {
            // 예외 발생 여부와 관계없이 항상 실행
            System.out.println("정리 작업 수행");
        }
        */
    }
}
